src/functions-and-arrays.js

```diff
@@ -1,19 +1,105 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers (num1, num2){
+  if(num1 > num2) {
+    return num1;
+  } else if (num1 < num2){
+    return num2;
+  } else {
+    return num1; 
+  }
+}
+
 // Iteration #2: Find longest word
+
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord (arrayOfWords){
+let longestWord = '';
+
+  for( let i = 0; i < arrayOfWords.length; i++){
+    if(longestWord.length === arrayOfWords.length){
+      continue;
```

Comment:
if(longestWord.length === arrayOfWords.length){
      continue;}
this condition is not necessary,  :)